packages/carousel/src/layoutToFlexWidths.js

```diff
@@ -3,7 +3,14 @@ export const layoutToFlexWidths = (layout, totalSlides) => {
     return []
   }
 
-  const layoutValues = layout?.split('-').map((i) => parseInt(i))
+  const layoutValues = layout?.split('-').map((i) => {
+    const value = parseInt(i)
+
+    // 33-33-33 is the only supported layout where values don't sum to 100. We need to
+    // account for this in order for slides not to be cut off.
+    return value === 33 ? value + 1 / 3 : value
```

Comment:
nb: can you put brackets around `(1 / 3)` just to make it clearer the order of operations

Reply:
I tried but the linter ate them

Reply:
`const oneThird = 1 / 3`? :trollface: